app/services/report_actions/save_report.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
```

Comment:
Why separate service?? You can simply create report elements within report_content.rb service.